.github/workflows/ci.yml

```diff
@@ -99,19 +99,24 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 20
-          cache: yarn
+
+      - name: Install pnpm
+        run: corepack enable
+
+      - name: Install pnpm
+        run: corepack enable
```

Comment:
This is duplicated.
Probably better to use `pnpm/action-setup` so it is possible to specify pnpm version.
And `actions/cache` needed to cache pnpm packages, so they are not downloaded every time.